### ISSUE TITLE
[Hosted Worlds] Meteor instantly impacts into the planet after being spawned in after removal despite the timer being set far from the day of impact

### ISSUE THREAD

**Issue by Ry4ndrew** [StrangeLoopGames/EcoIssues#21200]
Note: Test with both local world and cloud world

**Comment by j3fffff** [comment]
@WeaselDog can you advice on this one too

**Comment by Ry4ndrew** [comment]
While attempting to test #21125 when modifying with the DisasterPlugin by removing the meteor clicking Save and adding the meteor afterwards and clicking save will have the meteor impact the world upon entering a new world.

**Comment by j3fffff** [comment]
sorry i dont follow.
you are adding meteor second time with chat commands? and it impacts on a new world?

**Comment by Ry4ndrew** [comment]
![image](https://user-images.githubusercontent.com/49683923/113960853-adc6dd80-9857-11eb-9779-447fc4748b97.png)

I meant when starting a new world, the **CreateMeteor** by default is "On", if in an accident when starting a new world (which you have yet to log in) you were fiddling with the **DisasterPlugin**. You accidentally turned the **CreateMeteor** "Off" and then clicked **Save Configs** but you wanted it to be on, so you changed **CreateMeteror** to "On" and then click save **Save Configs**.

Upon joining the world, the meteor will impact instantly on Day 1.

Basically, any attempt to change the **CreateMeteor** from its "Off" state using the **DisasterPlugin** forces the meteor to impact the world.
